packages/vc-http-api/src/config/index.ts

```diff
@@ -1,5 +1,12 @@
 const config = {
   version: '0.0.0',
+  routes: {
+    disabled: [
+      'disabled',
```

Comment:
as I read this, there are currently no disabled routes

Reply:
correct - by default we leave everything on

Reply:
with example on how to disable